themes/schema.json

```diff
@@ -1685,6 +1686,38 @@
               }
             }
           }
+        },
+        {
+          "if": {
+            "properties": {
+              "type": { "const": "wifi" }
+            }
+          },
+          "then": {
+            "title": "WiFi Segment",
+            "description": "https://ohmyposh.dev/docs/wifi",
+            "properties": {
+              "properties": {
+                "properties": {
+                  "connected_icon": {
```

Comment:
we still need to remove the icons, right?

Reply:
and done